package.json

```diff
@@ -118,15 +118,23 @@
     "davidshimjs-qrcodejs": "0.0.2",
     "event-source-polyfill": "0.0.9",
     "highstock-release": "5.0.14",
-    "jquery": "3.2.1",
+    "iscroll": "5.2.0",
+    "jquery": "3.4.1",
     "jquery.scrollto": "2.1.2",
     "js-cookie": "2.2.0",
     "kinetic": "5.2.0",
+    "lodash.debounce": "4.0.8",
+    "mobx": "5.0.3",
+    "mobx-react": "5.4.2",
+    "mobx-utils": "5.4.0",
     "moment": "2.19.4",
     "promise-polyfill": "6.1.0",
     "prop-types": "15.6.1",
-    "react": "16.6.3",
-    "react-dom": "16.6.3",
+    "react": "16.8.6",
+    "react-dom": "16.8.6",
+    "react-pose": "4.0.8",
```

Comment:
afaik, we won't use react-pose, react-router-dom, and react-transition-group on binary static, that would be in deriv app

Reply:
cleaned